tests/sla_tests.js

```diff
@@ -231,7 +267,7 @@ describe('SLA Bridge and Channels Tester', function() {
     var sla = require('../lib/sla.js')(client, config, inbound, '42')
       .done();
 
-    bridges.push(getMockBridge({type: 'mixing', name: '42'}, function(){}));
+    bridges.push(getMockBridge({type: 'mixing', name: '42'}, function() {}));
```

Comment:
I'm not sure why there's a function being passed to getMockBridge. That function doesn't expect more than 1 parameter.